themes/reactos/static/css/style.blue.css

```diff
@@ -25,6 +25,14 @@
 	display: block !important;
 }
 
+html, body {
+	height: 100%;
+}
+
+.container-fluid {
```

Comment:
I don't like changing the behaviour of default Bootstrap classes

Reply:
Then what solution you propose? This is by far that works.

Reply:
I am with @Extravert-ir here. 
Please add an id or a class to the same div where .container-fluid is applying, so it looks something similar to: 
<div class="container-fluid main-content"> and apply the Flex change to the main-content instead. (My suggestion was a "how it could be done" change)
This way if we want to reuse container-fluid bootstrap class in any other divs we won't enforce a Flex attribute to them.
Thx!